lazy_tensor_core/lazy_tensor_core/core/lazy_model.py

```diff
@@ -973,3 +975,81 @@ def get_memory_info(device):
 
 def set_noop_execution_mode(enable):
     return lazy_tensor_core._LAZYC._ltc_set_noop_execution_mode(enable)
+
+
+def create_lazy_process_group(store: dist.Store, rank: int, size: int, timeout: timedelta):
+    """
+    Create a ProcessGroup that is dedicated to handle lazy devices for torch.distributed.
+
+    Args:
+        store (dist.Store): key/value store accessible to all workers, used
+                            to exchange connection/address information.
+        rank (int): Rank of the current process (it should be a
+                    number between 0 and ``size``-1).
+        size (int): Number of processes participating in the job.
+        timeout (timedelta): Timeout for operations executed against
+                             the process group.
+
+    Returns:
+        A new LazyProcessGroup instance.
+    """
+    return LazyProcessGroup(store, rank, size, timeout)
+
+
+class LazyProcessGroup(dist.ProcessGroup):
```

Comment:
would this class still be used after the fallback is replaced by tracing ops into lazy graphs?

Reply:
Well, I guess no. However they shouldn't conflict with each other. This is more like a fast way so we can have so sort of DDP capability in the upcoming release.